tests/puppeteer/pages/BO/shopParameters/general/maintenance/index.js

```diff
@@ -29,12 +32,44 @@ module.exports = class shopParamsMaintenance extends BOBasePage {
   /**
    * Enable / disable shop
    * @param toEnable, true to enable and false to disable
-   * @return {Promise<void>}
+   * @return {Promise<textContent>}
    */
   async changeShopStatus(toEnable = true) {
     if (toEnable) await this.page.click(this.enableShopLabel);
     else await this.page.click(this.disableShopLabel);
     await this.page.click(this.saveFormButton);
-    await this.page.waitForSelector(this.alertSuccessBloc);
+    return this.getTextContent(this.alertSuccessBloc);
+  }
+
+  /**
+   * Update Maintenance text
+   * @param text
+   * @return {Promise<textContent>}
```

Comment:
The docblock for `getTextContent` function is wrong.
it must be `@return string` and not `@return textContent` which is not a js type. (in commonPage.js)

```suggestion
   * @return {Promise<string>}
```